authui/src/authflowv2/components/modal.css

```diff
@@ -0,0 +1,56 @@
+@layer components {
+  :root {
+    --lockout__bg-color: var(--color-white);
+  }
+
+  .modal[open] {
+    @apply absolute;
+    @apply left-1/2;
+    @apply transform;
+    @apply -translate-x-1/2;
+
+    @apply flex;
+    @apply flex-col;
+    @apply items-center;
+    @apply justify-center;
+
+    @apply h-full;
+    @apply max-h-full;
+    @apply w-full;
+    @apply max-w-full;
```

Comment:
Having both
```
@apply absolute;
    @apply left-1/2;
    @apply transform;
    @apply -translate-x-1/2;
```

and 

``` 
@apply h-full;
    @apply max-h-full;
    @apply w-full;
    @apply max-w-full;
``` 
seems redundant. What are you going to achieve here? I saw you use absolute very often [like here](https://github.com/authgear/authgear-server/pull/3669/commits/c4f0b9973c2bdb55cb7cbbb6b3bf347261bdd640), while we have some better approaches.

If we want to make the modal cover the widget entirely, then we can just write

```
@apply absolute inset-0;
```

Also, putting relate properties on the same line improve readability, because this can tell the reader they are supposed to be effective together to achieve a certain effect.

Reply:
Was due to overriding dialog, updated base css to reset dialog style instead.